jobbergate_api/apps/users/routers.py

```diff
@@ -0,0 +1,48 @@
+import typing
+
+from asyncpg.exceptions import UniqueViolationError
```

Comment:
I don't have any really strong opinions about this but i've been having a lot of success with the `databases` package, which is related to the `fastapi` family of apps. It provides a unified API that you can use with multiple database backends. only advantage i can actually point to is that you can do development with sqlite on your desktop before deploying to postgres on the backend. it is async (in fact i think it just includes asyncpg when you install it).

that might be valuable to you, or it might not